cluster/userdata-master.yaml

```diff
@@ -337,7 +337,7 @@ write_files:
             limits:
               cpu: 200m
               memory: 1Gi
-        - image: registry.opensource.zalan.do/teapot/k8s-authnz-webhook:v0.1.4
+        - image: registry.opensource.zalan.do/teapot/k8s-authnz-webhook:v0.1.4-dirty-3
```

Comment:
It's just a placeholder, the new image will be ready as soon as the dependent PR is merged.